vocab/properties.json

```diff
@@ -584,22 +578,22 @@
         ]
     },
     "email": {
-        "description": null,
+        "description": "Means or system for transmitting messages electronically.",
```

Comment:
I think it is used more specifically to refer to an e-mail _address_

Reply:
Should we also be more specific in the property name? ("emailAddress")
For the description, how about "Address to which an electronic mail can be sent."

Reply:
I think you followed https://schema.org/email, where the property name is also only "email" (expecting an email address). An email address can be used for both sending and receiving emails. 

Maybe something like: "An address identifying an electronic mail box to which messages are delivered or sent from."